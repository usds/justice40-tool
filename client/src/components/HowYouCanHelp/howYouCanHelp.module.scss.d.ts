declare namespace HowYouCanHelpModuleScssNamespace {
  export interface IHowYouCanHelpModuleScss {
    howYouCanHelpContainer: string;
    listWrapper: string;
    howYouCanHelpList: string;
    howYouCanHelpListWrapper: string;
  }
}

declare const HowYouCanHelpModuleScssModule: HowYouCanHelpModuleScssNamespace.IHowYouCanHelpModuleScss & {
  /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
  locals: HowYouCanHelpModuleScssNamespace.IHowYouCanHelpModuleScss;
};

export = HowYouCanHelpModuleScssModule;
