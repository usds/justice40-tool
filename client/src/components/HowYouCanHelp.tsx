import React from 'react';
import * as styles from './HowYouCanHelp.module.scss';

const HowYouCanHelp = () => {
  return (
    <div className={styles.howYouCanHelpContainer}>
      <h2>How You Can Help Improve the Tool</h2>
      <ul>
        <li>If you have information that could help, weâ€™d love to hear from you.</li>
        <li>View our full set of data sources and methodology
            where you can add or download sources and check statuses on our data roadmap.</li>
        <li>Check out our timeline and send feedback or attend relevant events.</li>
        <li>Contact us and share the stories of your community.</li>
      </ul>
    </div>
  );
};

export default HowYouCanHelp;
