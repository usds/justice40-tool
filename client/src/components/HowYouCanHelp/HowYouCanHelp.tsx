import React from 'react';

import * as styles from './howYouCanHelp.module.scss';
import * as ABOUT_COPY from '../../data/copy/about';

const HowYouCanHelp = () => {
  return (
    <div className={styles.howYouCanHelpContainer}>
      <h2>
        {ABOUT_COPY.HOW_YOU_CAN_HELP_LIST_ITEMS.HEADING}
      </h2>
      <ul className={styles.howYouCanHelpListWrapper}>
        <li className={styles.howYouCanHelpList}>
          {ABOUT_COPY.HOW_YOU_CAN_HELP_LIST_ITEMS.LIST_ITEM_1}
        </li>
        <li className={styles.howYouCanHelpList}>
          {ABOUT_COPY.HOW_YOU_CAN_HELP_LIST_ITEMS.LIST_ITEM_2}
        </li>
        {/* <li className={styles.howYouCanHelpList}>
          {ABOUT_COPY.HOW_YOU_CAN_HELP_LIST_ITEMS.LIST_ITEM_3}
        </li> */}
        <li className={styles.howYouCanHelpList}>
          {ABOUT_COPY.HOW_YOU_CAN_HELP_LIST_ITEMS.LIST_ITEM_4}
        </li>
      </ul>
      <p>
        {ABOUT_COPY.HOW_YOU_CAN_HELP_LIST_ITEMS.PARA1}
      </p>
    </div>
  );
};

export default HowYouCanHelp;
