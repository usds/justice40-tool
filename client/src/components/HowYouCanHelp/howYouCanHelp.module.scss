@import "../utils.scss";

.howYouCanHelpContainer {
  margin: 2rem 0;
}

.howYouCanHelpList {
  list-style-type: disc;
  padding-bottom: 1rem;
}

.howYouCanHelpListWrapper {
  padding-left: 2rem;
}
