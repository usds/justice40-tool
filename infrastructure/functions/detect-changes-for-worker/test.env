REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1