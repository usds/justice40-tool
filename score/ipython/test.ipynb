{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
