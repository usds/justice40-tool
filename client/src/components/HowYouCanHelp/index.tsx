import HowYouCanHelp from './HowYouCanHelp';

export default HowYouCanHelp;
