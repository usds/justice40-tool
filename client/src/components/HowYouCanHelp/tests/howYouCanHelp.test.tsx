import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../../test/testHelpers';
import HowYouCanHelp from '../index';

describe('rendering of the HowYouCanHelp', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <HowYouCanHelp />
      </LocalizedComponent>,
  );

  it('checks if various text fields are visible', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
