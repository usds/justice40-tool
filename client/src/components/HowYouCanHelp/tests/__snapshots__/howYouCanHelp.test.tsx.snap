// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the HowYouCanHelp checks if various text fields are visible 1`] = `
<DocumentFragment>
  <div>
    <h2>
      How you can help improve the map
    </h2>
    <ul>
      <li>
        Provide 
        <a
          class="usa-link usa-link--external"
          data-cy=""
          href="https://www.surveymonkey.com/r/P3LWTSB"
          rel="noreferrer"
          target="_blank"
        >
          general feedback
        </a>
         on the CEJST website
      </li>
      <li>
        Suggest new 
        <a
          class="usa-link usa-link--external"
          data-cy=""
          href="https://www.surveymonkey.com/r/6G9TQJ8"
          rel="noreferrer"
          target="_blank"
        >
          data sources
        </a>
        .
      </li>
      <li>
        Any other questions? The best way to contact the Council on Environmental Quality (CEQ) is by filling out this 
        <a
          class="usa-link usa-link--external"
          data-cy=""
          href="https://www.surveymonkey.com/r/5LZ7MNB"
          rel="noreferrer"
          target="_blank"
        >
          form
        </a>
         . Otherwise, email: 
        <a
          class="usa-link"
          href="/en/methodology"
        >
          Screeningtool-Support@omb.eop.gov
        </a>
      </li>
    </ul>
    <p>
      
      The Council on Environmental Quality plans to issue a Request for Information in 2023. This will give the public time to use the tool before providing comments.    
    
    </p>
  </div>
</DocumentFragment>
`;
